### PR TITLE
Feat/use reflections hook migration

### PR DESCRIPTION
**Pull request by themonster2015**
## Checklist

- [ ] I have read the **README** document
- [ ] I have read the **CONTRIBUTING** document
- [ ] I have added tests to cover my changes
- [ ] All new and existing tests passed
- [ ] My commit message follows [Conventional Commit Guideline](https://conventionalcommits.org/)

### CODE REVIEW

File: extensions/apps/antenna/src/components/pages/entry-page/beam-page.tsx [hunk 1]
```diff
@@ -32,8 +31,8 @@ type BeamPageProps = {
 
 const BeamPage: React.FC<BeamPageProps> = props => {
   const { beamId, beamStream, beam } = props;
-  const { t } = useTranslation('app-antenna');
-  const { navigateToModal, getTranslationPlugin } = useRootComponentProps();
+  const { t } = useTranslation('app-akasha-integration');
```
Comment (didd) [inline extensions/apps/antenna/src/components/pages/entry-page/beam-page.tsx]:
```suggestion
  const { t } = useTranslation('app-antenna');
```

File: extensions/apps/antenna/src/components/pages/entry-page/reflection-page.tsx [hunk 2]
```diff
@@ -27,12 +27,11 @@ type ReflectionPageProps = {
 
 const ReflectionPage: React.FC<ReflectionPageProps> = props => {
   const { reflectionId, reflection, isLoggedIn } = props;
-  const { t } = useTranslation('app-antenna');
-  const { navigateToModal, getTranslationPlugin } = useRootComponentProps();
+  const { t } = useTranslation('app-akasha-integration');
```
Comment (didd) [inline extensions/apps/antenna/src/components/pages/entry-page/reflection-page.tsx]:
```suggestion
  const { t } = useTranslation('app-antenna');
```

File: extensions/apps/antenna/src/components/reflect-editor/index.tsx [hunk 3]
```diff
@@ -29,13 +28,12 @@ export type ReflectEditorProps = {
...
   showEditor: boolean;
-  pendingReflectionsVar: ReturnType<typeof createReactiveVar<ReflectEntryData[]>>;
   setShowEditor: (showEditor: boolean) => void;
 };
 
 const ReflectEditor: React.FC<ReflectEditorProps> = props => {
-  const { beamId, reflectToId, showEditor, pendingReflectionsVar, setShowEditor } = props;
-  const { t } = useTranslation('app-antenna');
+  const { beamId, reflectToId, showEditor, setShowEditor } = props;
+  const { t } = useTranslation('app-akasha-integration');
```
Comment (didd) [inline extensions/apps/antenna/src/components/reflect-editor/index.tsx]:
```suggestion
  const { t } = useTranslation('app-antenna');
```

File: extensions/apps/antenna/src/components/pages/entry-page/beam-page.tsx (hunk 1 above)
Comment (themonster2015) [reply extensions/apps/antenna/src/components/pages/entry-page/beam-page.tsx]: thanks Didd!

**Review by didd — APPROVED** [review]

**Pull request merged**